SU2_CFD/src/drivers/CMultizoneDriver.cpp

```diff
@@ -692,11 +663,9 @@ bool CMultizoneDriver::Monitor(unsigned long TimeIter){
       cout << "-------------------------------------------------------------------------" << endl;
     }
 
-    StopCalc = FinalTimeReached || MaxIterationsReached;
+    return (FinalTimeReached || MaxIterationsReached);
   }
 
-  return StopCalc;
```

Comment:
StopCalc is class data and was returned to itself, so I made the function to not use that var